src/applications/party_menu/unk_02083370.c

```diff
@@ -33,84 +33,84 @@
 
-static void sub_02083AD0(PartyMenuApplication *param0, int *param1);
-static void sub_0208472C(PartyMenuApplication *param0, int *param1);
-static void sub_02084808(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_MoveMon_Callback(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_Select1_Callback(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_FieldMove_Callback(PartyMenuApplication *param0, int *param1); // sub_02084808
 static void sub_02083E8C(PartyMenuApplication *param0, u8 param1);
 static void sub_02083FDC(PartyMenuApplication *param0, u8 param1, u8 param2);
 static void sub_02084134(PartyMenuApplication *param0);
 static void sub_02084420(PartyMenuApplication *param0, u8 param1);
-static void sub_020848A8(PartyMenuApplication *param0, int *param1);
-static void sub_020848C0(PartyMenuApplication *param0, int *param1);
-static void sub_020848D8(PartyMenuApplication *param0, int *param1);
-static void sub_020848F0(PartyMenuApplication *param0, int *param1);
-static void sub_02084908(PartyMenuApplication *param0, int *param1);
-static void sub_02084920(PartyMenuApplication *param0, int *param1);
-static void sub_02084938(PartyMenuApplication *param0, int *param1);
-static void sub_02084950(PartyMenuApplication *param0, int *param1);
-static void sub_02084968(PartyMenuApplication *param0, int *param1);
-static void sub_02084980(PartyMenuApplication *param0, int *param1);
-static void sub_02084998(PartyMenuApplication *param0, int *param1);
-static void sub_020849B0(PartyMenuApplication *param0, int *param1);
-static void sub_020849C8(PartyMenuApplication *param0, int *param1);
-static void sub_020849E0(PartyMenuApplication *param0, int *param1);
-static void sub_020849FC(PartyMenuApplication *param0, int *param1);
-static int sub_02084A18(PartyMenuApplication *param0);
+static void PartyMenu_FieldMove_Cut_Callback(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_FieldMove_Rock_Smash_Callback(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_FieldMove_Strength_Callback(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_FieldMove_Surf_Callback(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_FieldMove_Defog_Callback(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_FieldMove_Rock_Climb_Callback(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_FieldMove_Fly_Callback(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_FieldMove_Waterfall_Callback(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_FieldMove_Flash_Callback(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_FieldMove_Teleport_Callback(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_FieldMove_Dig_Callback(PartyMenuApplication *param0, int *param1);
+static void PartyMenu_FieldMove_Sweet_Scent_Callback(PartyMenuApplication *param0, int *param1);
```

Comment:
`PartyMenu` is described by the directory. If you think this file can reasonably be called `FieldMoves.c` then I think you can get away with just naming these something like `Callback_Surf`. Static functions don't need to worry as much about collisions.